crates/bevy_gltf/src/loader.rs

```diff
@@ -140,6 +140,11 @@ async fn load_gltf<'a, 'b>(
                 .map(|v| VertexAttributeValues::Float2(v.into_f32().collect()))
             {
                 mesh.set_attribute(Mesh::ATTRIBUTE_UV_0, vertex_attribute);
+            } else {
+                let len = mesh.count_vertices();
+                let uvs = vec![[0.0, 0.0]; len];
+                bevy_log::warn!("missing `TEXCOORD_0` vertex attribute, loading zeroed out UVs");
```

Comment:
Warnings (and to a lesser extent, info logs) in Bevy _must_ be actionable/fixable. But there will be cases where users won't be able to modify models (ex: they bought them on an asset store). I think we should consider "uv-less" models a "happy path" case.  

Can we make this a `debug` log instead?

Reply:
Done.